documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc

```diff
@@ -7,22 +7,20 @@
 = Enabling the example Grafana dashboards
 
 [role="_abstract"]
-Use Grafana dashboards to visualize Strimzi data.
-Add dashboards if you are using Prometheus and Grafana to extract metrics and monitor Strimzi.
-
-Strimzi provides xref:ref-metrics-config-files-{context}[example dashboard configuration files for Grafana].
-Example dashboards are provided in the `examples/metrics/grafana-dashboards` directory as JSON files:
-
-* `strimzi-kafka.json`
-* `strimzi-zookeeper.json`
-* `strimzi-operators.json`
-* `strimzi-kafka-connect.json`
-* `strimzi-kafka-mirror-maker-2.json`
-* `strimzi-kafka-bridge.json`
-* `strimzi-cruise-control.json`
-* `strimzi-kafka-exporter.json`
-
-The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
+Use Grafana to provide visualizations of Prometheus metrics.
+
+You can use your own Grafana deployment or deploy Grafana using the xref:assembly-metrics-config-files-{context}[example metrics configuration files] provided by Strimzi.
+The example files include a configuration file for a Grafana deployment
+
+* `examples/metrics/grafana-install/grafana.yaml`
+
+Strimzi also provides xref:ref-metrics-dashboards-{context}[example dashboard configuration files for Grafana] in JSON format.
+
+* `examples/metrics/grafana-dashboards`
```

Comment:
How does this file fit together with the `archive_metrics-dashboard-descriptions.adoc` file? There seems to be a big overlap.

Reply:
This file isn't in the guide anymore. If we're happy with the latest structure, I'll delete this file.